aggregator/src/recursion/circuit.rs

```diff
@@ -68,21 +76,39 @@ fn accumulate<'a>(
 
 #[derive(Clone)]
 pub struct RecursionCircuit<ST> {
+    /// The verifying key for the circuit.
     svk: Svk,
+    /// The default accumulator to initialise the circuit.
```

Comment:
It is the accumulator that recursion circuit begin with **when** the "previous" snark is not avaliable (aka. round = 0)